atomicapp/nulecule/container.py

```diff
@@ -90,7 +90,7 @@ def pull(self, image, update=False):
 
         cockpit_logger.info('Skipping pulling docker image: %s' % image)
 
-    def extract(self, image, source, dest, update=False):
+    def extract(self, image, source, dest, update=False, file=False):
```

Comment:
can you fpaste or pastebin what you have right now (the two new functions) and I'll try to understand your confusion and then try to help come up with a solution